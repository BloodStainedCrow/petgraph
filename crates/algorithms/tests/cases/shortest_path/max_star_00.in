version https://git-lfs.github.com/spec/v1
oid sha256:095e698480f5ca855e715fea307baa6a6414d384b517cd194dc77fda396ecdff
size 10833928
