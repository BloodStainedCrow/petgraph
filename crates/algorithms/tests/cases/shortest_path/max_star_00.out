version https://git-lfs.github.com/spec/v1
oid sha256:8e36190f48f36e4fb918417f71f25170b6c87c8177b90198d26252e00079c18e
size 81
